### DISCUSSION TITLE
After the datanode node in k8s is down, it cannot be restored

### DISCUSSION THREAD

**Discussion by lyn-my** [cubefs/cubefs#3277] [General]
![image](https://github.com/cubefs/cubefs/assets/47772644/f3eb8b66-ded6-4f8e-a47b-a2c46f42c425)

**Comment by leonrayang** [comment]
> ![image](https://private-user-images.githubusercontent.com/47772644/314471290-f3eb8b66-ded6-4f8e-a47b-a2c46f42c425.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0NTc5MzUsIm5iZiI6MTcxNDQ1NzYzNSwicGF0aCI6Ii80Nzc3MjY0NC8zMTQ0NzEyOTAtZjNlYjhiNjYtZGVkNi00ZjhlLWE0N2ItYTJjNDZmNDJjNDI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDMwVDA2MTM1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4NzdmNjkxYjRiMjdhZjY0YmQ1Y2Q1MzM0MGI0ZTY1ZTM1MWIyZDhkMTlhMzY3OGE3Y2U0NWFhOGNhNTQ5ZTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EfAWGZI6FdTywhtIRhIcKsPjoPHkhxJbiUh9SsydGh4)

@lyn-my May I know which version of the problem it is? Additionally, I need the logs of the following datanodes, including warnings and errors. The graph shows the warnings logs related to Raft.

**Comment by leonrayang** [comment]
/data/disk/datanode1/log/dataNode/output.log
/data/disk/datanode1/log/dataNode/dataNode_error.log
